MIGRATION.md

```diff
@@ -349,6 +349,8 @@ Like MongoSpecSupport, a number of helper functions are provided
 
 The underlying collection can be accessed via `repository.collection`.
 
+You can still use the `MongoSupport` trait directly to more closely replicate the old behaviour.
```

Comment:
Isn't this covered by the previous comment  "To refine this behaviour, you may use the composed traits directly."? Also seems to imply that previously only the `MongoSupport` trait from reactivemongo-test was used.

Maybe we should move it to it's own paragraph and rephrase it to something like:

"You may prefer to use the underlying traits directly to refine or replicate the old behaviour."

Reply:
Yeah good idea, it was more to stress that DefaultPlayMongoRepositorySupport was a new thing than brings together all the behaviours (including other traits), whereas MongoSupport is the direct successor to MongoSpecSupport